lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -253,6 +253,10 @@ def credit(money, payment_method, options = {})
         commit(order, :refund, options[:retry_logic], options[:trace_number])
       end
 
+      def store(creditcard, options = {})
+        authorize(0, creditcard, options.merge({ token_txn_type: 'GT' }))
+      end
```

Comment:
since the store process for Orbital happen if the token_txn_type is 'GT' I have added 'GT' as the default value because in this case it's a store transaction.
this is sended via options because is a GSF. and this should be setted by the user for a authorization transaction with 'UT' .
it's a good idea use a constant for this value, thanks